File: Python/Python104/coins.py
# coins = 0
# print("You have 0 coins.")
# response = (input("Do you want another? ")).lower
# if response == "yes":
#     coins += 1
#     print("You have %d coins." % coins)
# elif response == "no":
#     print("Bye")    
# else:
#     print("Good luck with that.")

coins = 0
print( "You have 0 coins." )
response = input("Would you like another? ").lower()

while response == "yes":
    coins += 1
    print("You have %d coins." % coins )
    response = input("Would you like another? ").lower() 
else:
    print("Bye then")

File: JavaScript/scripts/10-05/coins.js
// Write a program that will prompt you for how many coins you want. Initially you have no coins. It will ask you if you want a coin? If you type "yes", it will give you one coin, and print out the current tally. If you type no, it will stop the program
let coins = 0;
let more = `yes`; // prompt(`You have ${coins} coins. Do you want a coin? yes or no`);
while (more === `yes`) {
    more = prompt(`Do you want another? yes or no`);
    coins++;
    console.log(`You have ${coins} coins.`);
} 
console.log(`Byeeeeeeee`);



// let user = `yes`
// let coins = 0

// while (user === `yes`) {
//     user = prompt(`Want another?`);
//         coins++;
//         // debugger;
//         console.log(`You have ${coins} coins.`)
// }
// console.log(`Bye Felicia`)